core/src/test/java/org/apache/calcite/test/CalciteAssert.java

```diff
@@ -315,6 +305,7 @@ private Exception containsCorrectException(Throwable root) {
         assertThat(resultSetFormatter.string(), isLinux(expected));
       } catch (SQLException e) {
         throw new RuntimeException(e);
+        //  TestUtil.rethrow(e);
       }
```

Comment:
Why just add a comment ?

Reply:
That was a leftover as I tested forbiddenapis locally